pkg/errors/errors.go

```diff
@@ -441,6 +441,10 @@ func (e *ServiceError) IsFailedToCheckQuota() bool {
 	return e.Code == FailedToCheckQuota("").Code
 }
 
+func (e *ServiceError) IsNotFound() bool {
+	return e.Code == NotFound("").Code
```

Comment:
```suggestion
	return e.Code == ErrorNotFound
```

Reply:
Changed to use the existing `is404()` func.